addons/udes_priorities/views/priorities.xml

```diff
@@ -37,6 +37,7 @@
                 <field name="reference"/>
                 <field name="description"/>
                 <field name="picking_type_ids"/>
+                <field name="affected_pickings"/>
```

Comment:
Adding this to the view means we have to compute the field each time a priority is opened, which depending on the number of pickings could be quite expensive.

Maybe have a button instead to redirect to a view of a list of pickings that match the domain used in the compute field?

Might be worth getting a second opinion on this. :)

Reply:
So I think something like this should work (not tested at all, and just a suggestion so you can change as needed of course):

Add this to the top of the priority view:

```
<div class="oe_button_box" name="button_box">
    <button string="Outstanding Transfers"
        class="oe_stat_button"
        icon="fa-building-o"
        name="button_view_outstanding_pickings"
        type="object"
</div>
```

Add the method to the priority model:

```
def button_view_outstanding_pickings(self):
    self.ensure_one()
    return {
        "name": _("%s Outstanding Pickings") % self.name,
        "type": "ir.actions.act_window",
        "view_type": "tree",
        "view_mode": "tree",
        "res_model": "stock.picking",
        "target": "current",
        "domain": self._get_outstanding_pickings_domain, # use the method with the domain set, as per Miquel's suggestion
    }
```

Reply:
Resolving this as not needed as talking with Miquel.

If a user is told there are outstanding pickings then they can go to the picking view and find the pickings themselves.

So we don't need the list of pickings and can remove that many2many field.